spec/acceptance/reboot_pending_spec.rb

```diff
@@ -24,14 +24,14 @@
       }
       MANIFEST
     windows_agents.each do |agent|
-      apply_manifest_on agent, undo_pending_reboot_manifest
+      execute_manifest_on(agent, undo_pending_reboot_manifest)
     end
   end
 
   windows_agents.each do |agent|
     context "Agent #{agent}" do
       it 'Declare Reboot Required in the Registry' do
-        apply_manifest_on agent, pending_reboot_manifest
+        execute_manifest_on(agent, pending_reboot_manifest)
```

Comment:
This is strictly speaking a `before(:each)`, and doesn't really need `execute_manifest_on`